test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/IntegrationTests/CodeGenerationIntegrationTest.cs

```diff
@@ -263,233 +232,146 @@ public void IncompleteDirectives_DesignTime()
             // Assert
             AssertIRMatchesBaseline(document.GetIRDocument());
             AssertCSharpDocumentMatchesBaseline(document.GetCSharpDocument());
+            AssertDocumentCompiles(
+                document,
+                typeof(CodeGenerationIntegrationTest),
+                expectedErrors,
+                NEEDEDCLASSES);
```

Comment:
It would be nice to put this down near the bottom with the design-time equivalent.